test/support/chr/list_pagination.rb

```diff
@@ -74,16 +76,17 @@ def chage_position_to_middle_and_check_pagination(factory_name, class_name, list
     # If Module without disableFormCache need add reload page after "save"
     # Maybe not necessary
     test 'Change Position to the Middle and Check Pagination' do
+      skip "Waiting final answer from Travis CI support"
       create_per_page_plus_n_objects(2, factory_name)
       show_form_of_item(list_of_modules, @first_item.id)
       find('label.input-_position input').set(@middle_item._position-1)
       find_link('Save').click
       sleep(1.0)
-      assert page.has_css?('div.item-title', count: @loaded_items)
+      assert page.has_css?('div.item-title', count: @loaded_items), "Number of elements on page don't mach number of elements what must be loaded: Present: #{page.all('div.item-title').count}, Must be: #{@loaded_items}"
```

Comment:
Line is too long. [219/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.